src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -548,6 +550,8 @@ int AbortEnergyPlus(EnergyPlusData &state)
     CloseMiscOpenFiles(state);
     NumWarnings = fmt::to_string(state.dataErrTracking->TotalWarningErrors);
     NumSevere = fmt::to_string(state.dataErrTracking->TotalSevereErrors);
+    NumWarningsDuringInputs = fmt::to_string(state.dataErrTracking->TotalWarningErrorsDuringInputs);
```

Comment:
Tracking numerical values using strings is just nutty. Why are we doing this? For results framework? There has to be a better way.